src/Microsoft.SqlTools.ServiceLayer/EditData/EditSession.cs

```diff
@@ -378,5 +485,26 @@ public EditUpdateCellResult UpdateCell(long rowId, int columnId, string newValue
                 await errorHandler(e);
             }
         }
+
+        private string ConstructInitializeQuery()
+        {
+            // Using the columns we know, put together a query for the rows in the table
+            var columns = objectMetadata.Columns.Select(col => col.EscapedName);
+            var columnClause = string.Join(", ", columns);
+
+            return $"SELECT ${columnClause} FROM ${objectMetadata.EscapedMultipartName}";
+        }
+
+        public class EditSessionQueryExecutionState
```

Comment:
This is public and inside a public class, right? 2 questions:
- Should it be an inner class, or a standalone one?
- Can you document the class since it's public? Thanks!